hg-fast-export.py

```diff
@@ -443,6 +495,15 @@ def check_cache(filename, contents):
   	(revnode,_,_,_,_,_,_,_)=get_changeset(ui,repo,rev,authors)
   	mapping_cache[revnode.encode('hex_codec')] = str(rev)
 
+  if submodule_mappings:
+    # Make sure that all submodules are registerd in the submodule-mappings file
+    for rev in range(0,max):
+      ctx=revsymbol(repo,str(rev))
+      if (ctx.substate):
```

Comment:
The `()` around the expression is redundant and does not match the coding style/

Reply:
Will fix!